net/socket/sendmsg.c

```diff
@@ -93,19 +93,14 @@ ssize_t psock_sendmsg(FAR struct socket *psock, FAR struct msghdr *msg,
       return -EINVAL;
     }
 
-  if (msg->msg_iovlen != 1)
```

Comment:
This shouldn't actually be here. I initially wanted to use sendmsg() and extend it but I decided to just use sendto() instead.